CONTRIBUTING.md

```diff
@@ -43,17 +85,15 @@ Suppose for your PR you create a branch on `root.git`.
 Our CI infrastructure automatically picks up a branch with the same name in your fork of `roottest.git`
```

Comment:
Is this really clear? Would something like "The modified or added test can be in root or in roottest. Given the branch name in the ROOT fork used to make the PR,  our CI infrastructure automatically picks up the changes defined in the roottest PR which is made from an identical branch name, if it exists."